client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -21,22 +22,32 @@ import { recordTracksEvent } from 'state/analytics/actions';
 import FormattedHeader from 'components/formatted-header';
 import { checkInboundTransferStatus } from 'lib/domains';
 import support from 'lib/url/support';
+import paths from 'my-sites/domains/paths';
 
 class TransferDomainPrecheck extends React.PureComponent {
 	static propTypes = {
 		domain: PropTypes.string,
+		selectedSite: PropTypes.oneOfType( [ PropTypes.object, PropTypes.bool ] ),
 		setValid: PropTypes.func,
 		supportsPrivacy: PropTypes.bool,
 	};
 
 	state = {
-		unlocked: false,
-		privacy: false,
+		creationDate: '',
+		currentStep: 1,
+		daysTransferLocked: 60,
 		email: '',
+		expiryDate: '',
+		expiryDateNew: '',
+		expiryDateOk: true,
```

Comment:
Seems some error handling is needed, since when the server 500-ed, I got this:
<img width="740" alt="screen shot 2017-12-28 at 23 48 42" src="https://user-images.githubusercontent.com/3392497/34425464-78428bda-ec2c-11e7-80fe-eacb1c968b20.png">